ewatercycle/forcing/_hype.py

```diff
@@ -1,41 +1,35 @@
 """Forcing related functionality for hype"""
 
-from dataclasses import dataclass
 from pathlib import Path
 from typing import Optional
 
 from esmvalcore.experimental import get_recipe
 
-from .datasets import DATASETS
-from .default import DefaultForcing
 from ..util import get_time
-
-GENERATE_DOCS = """Hype does not have model specific options."""
-LOAD_DOCS = """Hype does not have model specific info."""
+from ._default import DefaultForcing
+from .datasets import DATASETS
 
 
-@dataclass
 class HypeForcing(DefaultForcing):
     """Container for hype forcing data."""
-
-    # Model-specific attributes (preferably with default values):
-    # ...
+    def __init__(
+        self,
+        start_time: str,
+        end_time: str,
+        directory: str,
+        shape: str,
+    ):
+        """
+            None: Hype does not have model-specific load options.
+        """
+        super().__init(start_time, end_time, directory, shape)
```

Comment:
```suggestion
        super().__init__(start_time, end_time, directory, shape)
```
Same for other forcing classes